wandb/sdk/wandb_run.py

```diff
@@ -1066,6 +1071,9 @@ def _set_run_obj(self, run_obj: RunRecord) -> None:
         self._run_obj = run_obj
         self._entity = run_obj.entity
         self._project = run_obj.project
+        self._settings.update(
```

Comment:
you might not need this change as I'm adding a new state to the settings that called `_apply_start` basically updates settings after the return form initial run

cc: @dmitryduev